packages/fxa-settings/src/components/CardHeader/index.test.tsx

```diff
@@ -35,7 +36,7 @@ describe('CardHeader', () => {
 
     expect(
       screen.getByRole('heading', {
-        name: `${MOCK_HEADING} to continue to account settings`,
+        name: `${MOCK_HEADING} to continue to ${MozServices.Default}`,
```

Comment:
The mocks file has `const MOCK_SERVICE_NAME = MozServices.FirefoxMonitor` and is shared with Storybook. I feel like we could go either way with setting it this way or not since in those mocks it's a real value. 🤔 

One other related thing to point out:
```
export const MOCK_DEFAULT_HEADING_FTL_ID = 'default-heading';
export const MOCK_CUSTOM_HEADING_FTL_ID = 'custom-heading';
```
I didn't think about this before but when we enable l10n tests we may need to tweak what we're doing here (unless we want to add these to `test.ftl` which could be fine), and think about Storybook as well since we will want bundles to show up there.